src/Menu.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+
+import { Box, IBoxProps } from './Box';
+import { Flex } from './Flex';
+import { Icon, IIcon } from './Icon';
+
+// TODO allow dividers in the menu
+// TODO allow custom renderMenu
+// TODO allow custom renderMenuItem
+// TODO allow custom renderTrigger to make this a popup type component
+
+/**
+ * MENU
+ */
+export interface IMenuProps {
+  menuItems: IMenuItemProps[];
+
+  attributes?: IBoxProps;
```

Comment:
Try to avoid used the styled util when possible. I tried something new with menu which I think could be a good convention. More "complex" components that use box as a base should take an attributes prop which can be used to override the default styling applied directly to box. This makes it a little cleaner by having us define the style directly in the component, and also separates the overridable style props from the important component ones like menuItems